BumbleBench doesn't build with JDK 12

### Problem Description

I'm trying to add BumbleBench targets for testing at Adopt as part of https://github.com/AdoptOpenJDK/openjdk-tests/issues/1379. BumbleBench compiles and runs successfully with JDK 8 and JDK 11 for both OpenJ9 and HotSpot. It doesn't compile with JDK 12, and hence the targets are just limited to 8 and 11 in the BumbleBench [playlist.xml](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/perf/bumbleBench/playlist.xml) file.

I believe we should see a similar issue with JDK 13 as well even though I haven't tested with it.

### Error
```
16:42:31  compile-all:
16:42:31      [javac] Compiling 8 source files to /java/jenkins/workspace/Piyush-Grinder/openjdk-tests/perf/bumbleBench/bumblebench
16:42:31      [javac] warning: [options] bootstrap class path not set in conjunction with -source 6
16:42:31      [javac] error: Source option 6 is no longer supported. Use 7 or later.
16:42:31      [javac] error: Target option 6 is no longer supported. Use 7 or later.
16:42:31  
16:42:31  BUILD FAILED
```

### Possible Solution

I believe, we'll need to update `source` and `target` for `javac` commands in https://github.com/AdoptOpenJDK/bumblebench/blob/master/build.xml.

### To Reproduce

- Remove the subsets constraint for any of the BumbleBench tests (https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/perf/bumbleBench/playlist.xml)
- Use the Adopt Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/build?delay=0sec
- Set the `JDK_VERSION` to 12.

--- a/build.xml
+++ b/build.xml
@@ -41,16 +41,16 @@ execute a build.
 		<javac srcdir="${src-core}" 
              destdir=""
              debug="on"
-             source="1.6"
-             target="1.6"
+             source="1.8"
+             target="1.8"
              includeantruntime="false"
              excludes="">
 		</javac>
 		<javac srcdir="${src-core}" 
              destdir=""
              debug="on"
-             source="1.7"
-             target="1.7"
+             source="1.8"
+             target="1.8"
              includeantruntime="false"
              includes="**/ParallelBench.java">
 		</javac>
@@ -59,8 +59,8 @@ execute a build.
 		<javac srcdir="${src-collections}" 
             debug="on" 
             destdir=""
-            source="1.6"
-            target="1.6" 
+            source="1.8"
+            target="1.8" 
             includeantruntime="false"
             classpath="${src-core}"
         	excludes="**/*ambda*.java">
@@ -80,8 +80,8 @@ execute a build.
 		<javac srcdir="${src-crypto}" 
             debug="on" 
             destdir=""
-            source="1.6"
-            target="1.6" 
+            source="1.8"
+            target="1.8" 
             includeantruntime="false"
             classpath="${src-core}">
 		</javac>
@@ -90,8 +90,8 @@ execute a build.
 		<javac srcdir="${src-examples}" 
             debug="on" 
             destdir=""
-            source="1.7"
-            target="1.7" 
+            source="1.8"
+            target="1.8" 
             includeantruntime="false"
             classpath="${src-examples}"
             includes="**/SocketEchoBench.java">
@@ -108,8 +108,8 @@ execute a build.
 		<javac srcdir="${src-examples}" 
             debug="on" 
             destdir=""
-            source="1.6"
-            target="1.6" 
+            source="1.8"
+            target="1.8" 
             includeantruntime="false"
             classpath="${src-examples}"
         	excludes="**/*ambda*.java, **/SocketEchoBench.java">
@@ -119,8 +119,8 @@ execute a build.
         <javac srcdir="${src-string}" 
             debug="on" 
             destdir=""
-            source="1.7"
-            target="1.7" 
+            source="1.8"
+            target="1.8" 
             includeantruntime="false"
             classpath="${src-string}">
         </javac>
@@ -168,8 +168,8 @@ execute a build.
 		<javac srcdir="${src-math}" 
              debug="on" 
              destdir=""
-             source="1.6"
-             target="1.6" 
+             source="1.8"
+             target="1.8" 
 			 includeantruntime="false"
              classpath="${src-math}">
 		</javac>

